frontend/src/components/main/Main.js

```diff
@@ -141,6 +128,64 @@ export default function Main(props) {
         }
         
       </div>
+      
+      
+      <div className="main-container">
+
+      <div className="element">
+          <label className="paste-label" class="fs-2" style={{textAlign: 'center', width: "100%"}}>Holdings</label>
+      </div>
+        <div className="element">
+            <label className="paste-label" class="fs-6" style={{textAlign: 'center', width: "100%"}}>Weth deposited: &nbsp;
+            <span style={{color: 'green'}}>{decimalFormat(vaultStore.totalAmounts.value[0], tokenStore.decimalsEth)} Weth</span></label>
+        </div>
+        <div className="element">
+            <label className="paste-label" class="fs-6" style={{textAlign: 'center', width: "100%"}}>Dai deposited: &nbsp;
+            <span style={{color: 'green'}}>{decimalFormat(vaultStore.totalAmounts.value[1], tokenStore.decimalsDai)} Dai</span></label>
+        </div>
+        <div className="element">
+            <label className="paste-label" class="fs-6" style={{textAlign: 'center', width: "100%"}}>Weth & Dai Ratio &nbsp;
+            <span style={{color: 'green'}}>{calculateRatio(decimalFormat(vaultStore.totalAmounts.value[0], tokenStore.decimalsEth), decimalFormat(vaultStore.totalAmounts.value[1], tokenStore.decimalsDai))}</span></label>
+        </div>
+        <div className="element">
+          <label className="paste-label" class="fs-6" style={{textAlign: 'center', width: "100%"}}>ETH/DAI Vault total shares: &nbsp;
+          <span style={{color: 'green'}}> {decimalFormat(vaultStore.totalSupply.value, vaultStore.decimals)}</span></label>
+        </div>
+
+        <div className="element">
+          <label className="paste-label" class="fs-2" style={{textAlign: 'center', width: "100%"}}>Deposit Cap</label>
+      </div>
+        <div className="element">
+            <label className="paste-label" class="fs-6" style={{textAlign: 'center', width: "100%"}}>% of the cap used: &nbsp;
+            <span style={{color: 'green'}}>{((vaultStore.totalSupply.value/vaultStore.maxTotalSupply.value) * 100).toFixed(2)}%</span></label>
```

Comment:
I'm not sure about this, @SBfin is `vault.totalSupply/vault.maxTotalSupply` the correct formula to get the % of cap used?

Reply:
yes it is the outstanding shares over the total shares